include/aws/cryptosdk/private/framefmt.h

```diff
@@ -33,6 +33,15 @@ struct aws_cryptosdk_frame {
     struct aws_byte_buf authtag;
 };
 
+#define MAX_PLAINTEXT_SIZE 65535
```

Comment:
Is there a better suited place to define an upper bound for plaintext size? Bounding the plaintext_size argument to serialize is important as mentioned further below.

Reply:
According to the Message Body Specification documents, I believe the upper bound should be 2^36 - 32, is that correct?

Reply:
There are three limits:
* Max frame size is 2^32 - 1
* Max frames is 2^32 - 1
* Max plaintext size **for unframed messages** is 2^36 - 32

Given that this is header concerns framed messages, only the first two are relevant here.

As for where these constants should be defined, or if they might already be (they should be...), but this header is definitely the wrong place.

Reply:
MAX_FRAME_SIZE seems to be defined in private/session.h, but maybe it would be good to have it in a more general header so it is accessible here as well.

From what I understand, this header concerns unframed messages as well, no? Since both serialize and deserialize call serde_nonframed depending on their arguments.

Reply:
@ttjsu-aws Where should we define these bounds?